app/instance/.env.example

```
@@ -18,6 +18,15 @@ SUPABASE_PASSWORD=
 SUPABASE_PORT=6543 #use Transaction shared pooler for db and session pooler for queue
 SUPABASE_DB_NAME=
 
+MONGO_SCHEME=mongodb
+MONGO_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGO_HOST key should go before the MONGO_SCHEME key

(UnorderedKey)